### PR TITLE
Sync build config

### PR DESCRIPTION
**Pull request by fhamon**
Take a look at http://jshint.com/docs/options/
for new options and what has been removed

### CODE REVIEW

File: workspace/assets/tasks/jshint.js [hunk 1]
```diff
@@ -47,7 +48,10 @@ module.exports = function jshint (grunt) {
 					App: true,
 					Loader: true,
 					_: true,
-					module: true
+					module: true,
+					google: true,
+					THREE: true,
+					TWEEN: true
```
Comment (nitriques) [inline workspace/assets/tasks/jshint.js]: Pending the validation of this namespace. cc  @PascalPiche

File: workspace/assets/tasks/jshint.js [hunk 2]
```diff
@@ -47,7 +48,8 @@ module.exports = function jshint (grunt) {
 					App: true,
 					Loader: true,
 					_: true,
-					module: true
+					module: true,
+					google: true
```
Comment (nitriques) [inline workspace/assets/tasks/jshint.js]: @PascalPiche wanted to add our `CSS3` module. Do you agree @fhamon ?

Comment (fhamon) [reply workspace/assets/tasks/jshint.js]: Sure !

**Comment by fhamon** [comment]
@nitriques take a look at this https://github.com/DeuxHuitHuit/symphonycms-template/pull/269/commits/bbe45c2cc13523c4042779be7e0a8be5fd18a568

**Pull request merged**